src/rpc-worker-loader.js

```diff
@@ -10,11 +10,14 @@ function workerSetup() {
 			else {
 				p = Promise.reject('No such method');
 			}
-			p.then(
-				result => {
+			p.then(result => {
 					postMessage({ type: 'RPC', id, result });
-				},
-				error => {
+				})
+				.catch(error => {
+					if (error.stack) {
+						postMessage({ type: 'RPC', id, error: { name: error.name, message: error.message, stack: error.stack } });
```

Comment:
For the sake of some byte-shaving, could we just send `error: error.stack` or `error: String(error)` here?

Reply:
Ok I will have a look soon

Reply:
@developit Ok, I have some time to have a look at your comment.

The reason I initially passed:

`error: { name: error.name, message: error.message, stack: error.stack }` is because I want to be able to fully recreate the Error on the other side (i.e. [here](https://github.com/developit/workerize-loader/pull/33/files#diff-79163d4a3035efb533c9cc52b8d8857eR12))

I'm trying `String(error)` to see if it still works.

Reply:
I didn't have the time to test it, but I think you won't get the full stack from inside the worker if you don't pass all `{ name: error.name, message: error.message, stack: error.stack }` can you accept it like this? I've been using this approach in various cases and it works fine! 🙏

Reply:
When we have the time later.. we can improve it again, I have other improvements to suggest too, thanks!